salt/utils/icinga2.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+'''
+Icinga2 Common Utils
+=================
+
+This module provides common functionality for icinga2 module and state.
+
+.. versionadded:: 2018.8.3
+'''
+
+# Import python libs
+import logging
+import subprocess
+import re
+
+# Import Salt libs
+import salt.utils.path
+
+log = logging.getLogger(__name__)
+
+
+def execute(cmd, ret_code=False):
```

Comment:
WIth this, would it perhaps be better just to use Salt's `cmd.run` or `cmd.retcode` module functions that already exist?

Reply:
Will see if cmd.retcode works, I just realized I did not test that one.

Reply:
Thanks. If we can figure out the exact issue with the `cmd` module, it would be great to have an issue filed against that so we can get it fixed as well. Thanks.